Source/WebKit/Shared/wpe/WebEventFactoryWPE.cpp

```diff
@@ -133,16 +133,21 @@ WebMouseEvent WebEventFactory::createWebMouseEvent(WPEEvent* event)
     auto button = WebMouseEventButton::None;
     auto position = positionFromEvent(event);
     unsigned clickCount = 0;
+    WebMouseEventSyntheticClickType syntheticClickType = WebMouseEventSyntheticClickType::NoTap;
```

Comment:
Not really important, but in this particular case I find `auto` more readable because it is clear what will be the type of the variable, given that the type of the `enum` is used on the right-hand side of the assignment:

```suggestion
    auto syntheticClickType = WebMouseEventSyntheticClickType::NoTap;
```